EVGEN/AliGenReadersEMD.h

```diff
@@ -33,13 +33,15 @@ class AliGenReadersEMD : public AliGenReader
     void TrackAll() {fPcToTrack = kAll;}
     void SetStartEvent(Int_t nev) {fStartEvent = nev;}
     void SetNtupleName(TString s) {fNtupleName=s;}
+    void SetInvertPz() {fInvertPz=kTRUE;}
```

Comment:
```suggestion
    void SetInvertPz(Bool_t invert = kTrue) {fInvertPz=invert;}
    Bool_t GetInvertPz() const { return fInvertPz; }
```

Reply:
Hi @qgp,

thanks for this suggestion. Changed accordingly.

Cheers,
Michael